src/main/java/org/sunbird/portal/department/model/Actor.java

```diff
@@ -0,0 +1,59 @@
+package org.sunbird.portal.department.model;
+
+import javax.annotation.Generated;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+"id",
+"type"
+})
+
+public class Actor {
+
+@JsonProperty("id")
+private String id;
+@JsonProperty("type")
+private String type;
+
+/**
+* No args constructor for use in serialization
+*
+*/
+public Actor() {
+}
+
+/**
+*
+* @param id
+* @param type
+*/
+public Actor(String id, String type) {
```

Comment:
Need to move this to another new package which should be separate for telemetry related classes.